doordeck-sdk/src/androidMain/kotlin/com/doordeck/multiplatform/sdk/api/PlatformApi.android.kt

```diff
@@ -23,340 +26,445 @@ actual object PlatformApi {
      * @see PlatformClient.createApplicationRequest
      */
     @DoordeckOnly
-    suspend fun createApplication(application: PlatformOperations.CreateApplication) {
-        return PlatformClient.createApplicationRequest(application.toBasicCreateApplication())
-    }
+    suspend fun createApplication(application: PlatformOperations.CreateApplication) = PlatformClient
+        .createApplicationRequest(application.toBasicCreateApplication())
 
     /**
      * Async variant of [PlatformApi.createApplication] returning [CompletableFuture].
      */
     @DoordeckOnly
-    fun createApplicationAsync(application: PlatformOperations.CreateApplication): CompletableFuture<Unit> {
-        return completableFuture { createApplication(application) }
+    fun createApplicationAsync(
+        application: PlatformOperations.CreateApplication
+    ): CompletableFuture<Unit> = completableFuture {
```

Comment:
I think create application returns a Location header which contains the UUID of the application created, we could return the UUID part by extracting it from that URL, maybe a task for another time.